data-table/__tests__/table.js

```diff
@@ -1,9 +1,65 @@
+// test 4-5 function
```

Comment:
This shouldn't be here - or at least, it should be descriptive for the entire file, not just what's in this PR